src/app/frontend/deploy/createnamespace_dialog.js

```diff
@@ -28,7 +28,23 @@ export default function showNamespaceDialog(mdDialog, event, namespaces) {
     controllerAs: 'ctrl',
     clickOutsideToClose: true,
     targetEvent: event,
-    templateUrl: '/deploy/createnamespace.html',
+    template: `<md-dialog aria-label="Create a new namespace" layout="column" layout-padding>
```

Comment:
Leave the html template file and just change `/deploy/createnamespace.html` to `deploy/createnamespace.html`. This will work.